content/en/armory-enterprise/armory-admin/policy-engine/policy-engine-enable/policy-engine-plug-enable.md

```diff
@@ -17,187 +17,25 @@ Make sure the following requirements are met:
 
 ## Setup
 
-The Policy Engine Plugin can be enabled using one of the following methods:
+The Policy Engine Plugin can be enabled using the Armory Operator or Halyard.
 
-1. Docker image as an init container on each affected service
-
-1. Using a remote plugin repository
-
-### Docker image as init container
 
 {{< tabs name="enable-plugin" >}}
 {{% tabbody name="Operator" %}}
 
 You can use the sample configuration to install the plugin, but keep the following in mind:
 
-- The `patchesStrategicMerge` section for each service is unique. Do not reuse the snippet from one service for the other services.
-
-- Make sure to replace `<PLUGIN_VERSION>` with the version of the plugin you want to use. For a list of versions, see [Release notes](#release-notes).
+- Make sure to replace the version number listed after `&version` with the version of the plugin you want to use. For a list of versions, see [Release notes](#release-notes).
 
 - This configuration must go into `spinnakerservice.yml`. It cannot be patched in through Kustomize.
```

Comment:
Is this still true?

Reply:
I think that statement is... confusing and not needed....